Appsilon/rhino-showcase
# Feat: Use `logic` and `view` structure

### Changes
Closes #19 

### How to test
* Pull this branch.
* Run `renv::restore(clean = TRUE)`.
* Run app.
* App should work.

### Notes
* All scripts with a Shiny module, I moved to `app/view`.
* All scripts without a Shiny module, I moved to `app/logic`.
* `utils.R` uses some Shiny functions and elements, but since it does not contain a Shiny module I moved it to `app/logic`.
